packages/browser/src/__tests__/autocapture.test.ts

```diff
@@ -866,6 +882,62 @@ describe('Autocapture system', () => {
             expect(beforeSendMock.mock.calls[0][0].properties['$elements'][0]).not.toHaveProperty('attr__href')
         })
 
+        it('captures href from anchors with IDs containing sensitive substrings like password', () => {
+            const elTarget = document.createElement('span')
+            const elAnchor = document.createElement('a')
+            elAnchor.appendChild(elTarget)
+            elAnchor.setAttribute('id', 'password')
```

Comment:
this probably would have failed before because of the checks inside `shouldCaptureElement` on 'sensitive names'